run_linux.go

```diff
@@ -2403,6 +2421,20 @@ func (b *Builder) runSetupRunMounts(mounts []string, secrets map[string]string,
 	return finalMounts, artifacts, nil
 }
 
+func (b *Builder) getBindMount(tokens []string, rootUID, rootGID, processUID, processGID int) (*spec.Mount, error) {
+	var optionMounts []specs.Mount
+	mount, err := parse.GetBindMount(tokens, b.ContextDir)
```

Comment:
Use the value from `RunOptions.ContextDir` here, and return an error from here if one hasn't been set.